### DISCUSSION TITLE
Auto-version server in 1.0

### DISCUSSION THREAD

**Discussion by dcecile** [hyperium/hyper#3228] [Q&A] [answered]
https://github.com/hyperium/hyper/pull/3013 has a note to check `hyper-util` for the auto-version server feature, but I couldn't find it. Could I get a quick pointer?

Alternatively, is it common to run both HTTP/1.1 and HTTP/2 servers separately...? Or are folks opting to just serve HTTP/2?

**Comment by seanmonstar** [comment, marked as answer]
It's not merged yet, but here is a pull request in progress: https://github.com/hyperium/hyper-util/pull/11

**Reply by dcecile** [reply]
Got it, thank you!
